### DISCUSSION TITLE
how to set the airflow direction

### DISCUSSION THREAD

**Discussion by yuyachannpo** [firemodels/fds#13808] [Q&A]
Hello everyone,

I would like to simulate the air distribution in a room using the following FDS Codes.
But I am unsure how to set the airflow direction.  Could you please give me some help me?
&SURF ID='BLOWER', VEL=-5, TMP_FRONT=50. /
&OBST XB= 0.0, 0.2,3.2, 3.6, 2.0, 2.2 /
&VENT XB= 0.2, 0.2,3.2, 3.6, 2.0, 2.2,SURF_ID='BLOWER',COLOR='RED' /

Thank you very much.

Best regards,

**Comment by drjfloyd** [comment]
Have you tried looking to see what the User's Guide has to say?

**Comment by yuyachannpo** [comment]
Thank you very much.
By referring to the user guide, I managed to set the direction of the airflow.

Best regards,
